examples/classic/distributed/ewd840/ewd840.qnt

```diff
@@ -115,109 +104,100 @@ action Deactivate(i) = all {
   tcolor'    = tcolor,
 }
 
-/***************************************************************************)
-(* actions performed by the underlying algorithm                           *)
-(***************************************************************************/
+/// Actions performed by the underlying algorithm
 action Environment =
-  val i = oneOf(Node)
+  nondet i = oneOf(Nodes)
   any {
     SendMsg(i),
     Deactivate(i),
   }
 
-/***************************************************************************)
-(* next-state relation: disjunction of above actions                       *)
-(***************************************************************************/
-action Next = any {
+/// Next-state relation: disjunction of above actions
+action step = any {
   System,
   Environment,
 }
 
 val vars = (active, color, tpos, tcolor)
 
-// temporal Spec = Init and always(Next.orKeep(vars) and System.weakFair(vars))
+// temporal Spec = init and always(step.orKeep(vars) and System.weakFair(vars))
```

Comment:
I assume we already had this commented out because of `init`?
Probably good to treat it the same as below (i.e., just comment out `init`, or whatever we decide to do there)